ui/src/views/PageNotFoundView.vue

```diff
@@ -21,11 +26,44 @@
 </template>
 
 <script>
-  export default { name: "ErrorView" };
+  export default { name: "PageNotFoundView",
+  computed: {
+    mainPicture() {
+      return require("@assets/svg/404.png");
+    }
+  } };
 </script>
 
 <style lang="scss">
-  #error-view h1.text-h1 {
-    font-size: 14rem !important
+  #error-view-content {
```

Comment:
Targeting elements with IDs is not a good approach. Not just that they are not reusable, it may cause conflicts later since IDs have stronger 'power' over classes. 
html tag > id > class

the best approach is just to use classes.

Reply:
I will change this page to use the content card component.

Reply:
Alright, thank you!